arangod/Indexes/IndexFactory.cpp

```diff
@@ -76,6 +76,30 @@ struct InvalidIndexFactory : public IndexTypeFactory {
 }  // namespace
 
 namespace arangodb {
+namespace helpers {
+
+IndexId extractId(velocypack::Slice slice) noexcept {
+  auto id = IndexId::none();
+  auto value = slice.get(StaticStrings::IndexId);
+  if (value.isNumber()) {
+    id = IndexId{value.getNumber<IndexId::BaseType>()};
+  } else if (value.isString()) {
+    IndexId::BaseType base;
+    if (absl::SimpleAtoi(value.stringView(), &base)) {
```

Comment:
absl used here without the appropriate header being included. I suggest we add the proper include to the .cpp file.